frame/contracts/src/benchmarking/mod.rs

```diff
@@ -109,21 +109,18 @@ where
 		endowment: Endow,
 	) -> Result<Contract<T>, &'static str>
 	{
-		use sp_runtime::traits::{CheckedDiv, SaturatedConversion};
 		let (storage_size, endowment) = match endowment {
 			Endow::CollectRent => {
 				// storage_size cannot be zero because otherwise a contract that is just above
 				// the subsistence threshold does not pay rent given a large enough subsistence
 				// threshold. But we need rent payments to occur in order to benchmark for worst cases.
-				let storage_size = ConfigCache::<T>::subsistence_threshold_uncached()
-					.checked_div(&T::DepositPerStorageByte::get())
-					.unwrap_or_else(Zero::zero);
+				let storage_size = u32::max_value() / 10;
```

Comment:
I am a bit clueless about this change, but I don't really know this part of the code.

Reply:
I guess so. The benchmarks where broken and this new parameterization fixed it. Most probably it was broken by a previous PR where I did not run the benchmarks.